.github/workflows/python-linters.yml

```diff
@@ -0,0 +1,25 @@
+# For most recent version see:
+# https://github.com/WIPACrepo/wipac-dev-tools/blob/main/.github/workflows/python-linters.yml
+# Copy any updates to wipac-dev-tools.
+
+name: Python Linters
+
+on: [push]
+
+jobs:
+
+  flake8:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v2
+      - run: pip install flake8
+      - run: flake8 . --ignore=E203,E231,E226,E501,W503,W504
```

Comment:
Do we want to manually list the error codes here, or specify them in `setup.cfg`?
Also, do we want to lint the tests as well, or exclude them (or does it do that automatically?)

Reply:
I'd rather they be here. It makes copy-pasting much simpler

Reply:
As for the tests, they'll be linted because they're in the tree and .py files. I think that's fine, since tests are part of the codebase

Reply:
To elaborate on specifying flake8 flags in `setup.cfg`, this could be the "project" spec while the action yaml would be the _dev team's spec_. `setup.cfg`'s ignore flags would be a subset of the team's.